src/components/CombinedFeaturePanel/components/Gallery/Gallery.tsx

```diff
@@ -129,7 +136,7 @@ export const Gallery: FC<{
   const handleOnCloseAutoFocus = (event: Event) => {
     event.preventDefault();
     document
-      .querySelector<HTMLAnchorElement>(`[data-image-id=${activeImage?._id}]`)
+      .querySelector<HTMLAnchorElement>(`[data-image-id="${activeImage?._id}"]`)
```

Comment:
I stumbled upon another bug when testing this and I thought it would be okay to fix it in this PR as well. When the activeImageId starts with a numeric character, quotes are needed.